tests/api/benchmark/test_permission_group.py

```diff
@@ -114,7 +120,12 @@ def test_permission_group_update(
         "id": graphene.Node.to_global_id("Group", group.id),
         "input": {
             "name": "New permission group",
-            "permissions": [AccountPermissions.MANAGE_SERVICE_ACCOUNTS.name],
+            "addPermissions": [AccountPermissions.MANAGE_SERVICE_ACCOUNTS.name],
+            "removePermissions": [AccountPermissions.MANAGE_USERS.name],
+            "addUsers": [graphene.Node.to_global_id("User", staff_user.pk)],
+            "removeUsers": [
+                graphene.Node.to_global_id("User", group.user_set.first().pk)
+            ],
```

Comment:
Could use some comments or query string to explain what we are doing and testing

Reply:
I am just testing `PermissionGroupUpdate` mutation, I think that's clear.

Reply:
I mean, explaining what's going on, that you are ensuring it correctly adds the 'manage service accounts' permission and drops the 'manage users' permissions and removes add the default staff user into the group and then remove the <???> user from the group.

A docstring explaining what is going on that wall of code

Reply:
But it's a benchmark test. There is simple logic in these tests, just uses all available input field. As far as I know, the aim of the benchmark tests is just checking if the number of queries doesn't change. I really don't see any reason to add a comment here.